sdk/src/Core/overrides.targets

```diff
@@ -0,0 +1,5 @@
+<!--
+ this is a placeholder file to override the System.Text.Json version for netstandard20 and net472 targets in the build system.
+-->
+<Project>
```

Comment:
I'm unfamiliar on how having this empty file makes so the build system includes version 8.0.5. Can you give me the brief description on how this is used?

Reply:
^^ yup, i did add a bit more context though